frame/executive/src/lib.rs

```diff
@@ -320,7 +320,7 @@ where
 	) {
 		extrinsics.into_iter().for_each(|e| if let Err(e) = Self::apply_extrinsic(e) {
 			let err: &'static str = e.into();
-			panic!(err)
+			panic!("{}", err)
```

Comment:
Doesn't the error implement `Display` itself?

Reply:
yeah only on "std", I guess we can implement display on on not(std) as well